packages/components/image-viewer/__tests__/image-viewer.test.tsx

```diff
@@ -1,24 +1,18 @@
 import { nextTick } from 'vue'
+import { mount } from '@vue/test-utils'
 import { describe, expect, test } from 'vitest'
-
 import { IMAGE_SUCCESS } from '@element-plus/test-utils/mock'
-import makeMount from '@element-plus/test-utils/make-mount'
 import ImageViewer from '../src/image-viewer.vue'
 
-const mount = makeMount(ImageViewer, {
-  props: {
-    urlList: [IMAGE_SUCCESS],
-  },
-})
-
 async function doubleWait() {
   await nextTick()
   await nextTick()
 }
 
 describe('<image-viewer />', () => {
   test('big image preview', async () => {
-    const wrapper = mount()
+    const wrapper = mount(<ImageViewer urlList={[IMAGE_SUCCESS]} />)
```

Comment:
## Suggestion 

How about using `mount(() =>)` syntax to maintain consistency with other existing test codes?

Reply:
No, both ways make sense. In this case, passing object component is to make it easier to use the methods on the component. If I pass functional component, I have to use `wrapper.findComponent(ImageViewer).emitted('close')` instead of `wrapper.emitted('close')` (line#20)

Reply:
Ok!